frontend/packages/ux-editor/src/components/config/componentSpecificContent/AttachmentList/AttachmentListContent.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Combobox } from '@digdir/design-system-react';
+import { useTranslation } from 'react-i18next';
+import type { IGenericEditComponent } from '../../componentConfig';
+import classes from './AttachmentListContent.module.css';
+
+type IAttachmentListContent = IGenericEditComponent & {
+  selectedAttachments: string[];
+  attachments: string[];
+  onlyCurrentTask: boolean;
+  includePdf: boolean;
+};
+
+export const AttachmentListContent = ({
+  component,
+  handleComponentChange,
+  selectedAttachments,
+  attachments,
+  onlyCurrentTask,
+  includePdf,
+}: IAttachmentListContent) => {
+  const { t } = useTranslation();
+
+  const handleValueChanges = (updatedSelection: string[]) => {
+    const lastSelected = updatedSelection[updatedSelection.length - 1];
+
+    updatedSelection =
+      lastSelected === 'include-all'
+        ? ['include-all']
+        : updatedSelection.filter((dataType) => dataType !== 'include-all');
```

Comment:
Er det slik at `include-all` betyr at alle alternativene skal velges? I så fall ville jeg vurdert å heller lage en egen knapp/avkrysningboks som markerer alle alternativene. Da blir det mer tydelig for brukeren at alt er valgt, i tillegg til at det blir færre klikk hvis man f.eks. vil velge alle alternativene unntatt ett. Hva tenker du om det?